features/dd-sdk-android-rum/src/main/json/rum/_common-schema.json

```diff
@@ -41,6 +41,11 @@
       "description": "The build version for this application",
       "readOnly": true
     },
+    "build_id": {
```

Comment:
let's remove this one, this https://github.com/DataDog/dd-sdk-android/pull/1756 covers it

Reply:
I'll wait and rebase once #1756 goes in.  I can work off of this branch to start work on the source override.